.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -16,4 +16,23 @@ Fixes #?
 Documentation
 - [ ] Does the registry operator documentation need to be updated with your changes?
 
-**How to test changes / Special notes to the reviewer**:
+<!--
+Instructions for locally testing changes made to the operator, drawn from CONTRIBUTING.md
+-->
+**Testing changes**
+
+Unit Tests:
+- Run `make test` from root of the repository.
+
+<!--
+Will need to be updated after the completion of https://github.com/devfile/api/issues/1523
+-->
+Integration Tests:
```

Comment:
looks good to me and very useful info I have to say.

I was wondering if it makes sense to:
* Make the integration tests instructions a section in contributing.md
* Replace the text here with a link to contributing.md

This way we only have to update contributing guide if we want to make any change. WDYT?

Reply:
+1 on @thepetk suggestion, @Jdubrick we can the link to the Integration Tests section under `CONTRIBUTING.md` which also contains your changes, similar to what you did in the README.

Reply:
Good idea thanks for catching that!